modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/validation/ExclusiveParameterModelValidator.java

```diff
@@ -106,6 +109,9 @@ protected void defaultVisit(MetadataType metadataType) {
 
           @Override
           public void visitSimpleType(SimpleType simpleType) {}
+
+          @Override
+          public void visitObject(ObjectType objectType) {}
```

Comment:
The idea was throwing an IllegalModelDefinitionException for all the different MetadataTypes except for these 2 situations.